packages/tests/server/httpSubscriptionLink.test.ts

```
@@ -31,16 +31,37 @@ import { zAsyncIterable } from './zAsyncIterable';
 
 const sleep = (ms = 1) => new Promise((resolve) => setTimeout(resolve, ms));
 
+const returnDataSymbol = Symbol('returnData');
+
+export interface MyEvents {
+  data: (str: Error | number | [typeof returnDataSymbol, number]) => void;
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)